dask_expr/tests/test_collection.py

```diff
@@ -609,6 +609,7 @@ def test_to_timestamp(pdf, how):
         lambda df: df.astype(int),
         lambda df: df.clip(lower=10, upper=50),
         lambda df: df.x.clip(lower=10, upper=50),
+        lambda df: df.clip(lower=3, upper=7, axis=1),
```

Comment:
Do we want `None` and `0` cases for axis?

Reply:
Dask/dask has those, so not super concerned